spec/requests/users/users_show_spec.rb

```diff
@@ -3,8 +3,28 @@
 RSpec.describe 'Show user', type: :request do
   let(:user) { FactoryBot.create :user }
 
+  before { sign_in user }
+
   it 'returns http success' do
```

Comment:
I also thought about this, but many contexts already provide the right set up of notes and attributes etc. so this would probably introduce more duplication than a single `sign_in` statement ...

Reply:
You're right, a little bit more code for a better understanding of the tests, but this is so small it should be understandable like this too :+1: